app/experimenter/experiments/api/v5/serializers.py

```diff
@@ -298,26 +245,27 @@ def update(self, experiment, data):
         return experiment
 
 
-class NimbusExperimentUpdateSerializer(
-    NimbusChangeLogMixin,
-    NimbusStatusRestrictionMixin,
-    NimbusExperimentOverviewMixin,
-    NimbusExperimentDocumentationLinkMixin,
+class NimbusExperimentSerializer(
```

Comment:
Does this need renaming? We've got a serializer for in-progress updates and one for the final review state - I'd think that the ready-for-review serializer would fit the name better but 🤷

Reply:
Well I just renamed it because it used to be used only for updating and there was a separate one for creating, but now this one is used for creating and updating so I thought it didn't make sense to keep calling it update?